src/monero_ux_nano.c

```diff
@@ -673,10 +646,14 @@ void ui_menu_any_pubaddr_display(unsigned int value __attribute__((unused)),
             break;
     }
 
-    monero_base58_public_key(G_monero_vstate.ux_address, pub_view, pub_spend, is_subbadress,
-                             paymanetID);
+    int error = monero_base58_public_key(G_monero_vstate.ux_address, pub_view, pub_spend,
+                                         is_subbadress, paymanetID);
+    if (error) {
+        return error;
+    }
```

Comment:
return value is not handle by function that call this.
Maybe monero_base58 could be done before?